packages/popover/src/vaadin-popover.js

```diff
@@ -131,10 +155,19 @@ class Popover extends PopoverPositionMixin(
     this._overlayElement = this.shadowRoot.querySelector('vaadin-popover-overlay');
   }
 
+  /** @protected */
+  connectedCallback() {
+    super.connectedCallback();
+
+    document.addEventListener('click', this.__onGlobalClick);
```

Comment:
The overlay's global click listener is configured to catch clicks during the capturing phase. Should it be the same here?

Reply:
Done.